### PR TITLE
Alert Component

### PR DESCRIPTION
**Pull request by Mahi24-prog**
### Types 

- Simple Alert
- Outline Alert

### Review Link 
https://alertui.netlify.app/

### CODE REVIEW

**Review by Mahi24-prog — COMMENTED** [review]
Review please

File: component/alert.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+.alert{
+    padding: 1.5rem;
+    border-radius: 1rem;
+}
+
+/* Simple Alert  */
+.alert-primary{
+    background-color: #cce5ff;
+    color: 	#0275d8;
```
Comment (sakleprajwal) [inline component/alert.css]: Avoid unnecessary spaces

**Review by sakleprajwal — COMMENTED** [review]
Suggested minor changes.

File: component/alert.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+.alert-secondary{
+    background-color: #e2e3e5;
+    color: #576574;
+}
+
+.alert-success{
```
Comment (prathmeshjagtap) [inline component/alert.css:37]: Add variable naming to all colours , everything else looks good

**Review by prathmeshjagtap — CHANGES_REQUESTED** [review]

File: component/alert.css (hunk 2 above)
Comment (Mahi24-prog) [reply component/alert.css:37]: ok

File: component/alert.css (hunk 1 above)
Comment (Mahi24-prog) [reply component/alert.css]: ok

File: component/alert.css (hunk 2 above)
Comment (Mahi24-prog) [reply component/alert.css:37]: variables added for color

**Pull request merged**